modules/planning/conf/planning.conf

```diff
@@ -14,6 +14,8 @@
 --default_cruise_speed=20.00
 --enable_scenario_side_pass
 --enable_scenario_traffic_light
+--enable_collision_detection
```

Comment:
@YajiaZhang This flag has been deprecated via commit https://github.com/ApolloAuto/apollo/commit/1eff2859c13803037d62599cb14415a0f6b4f066  Please change accordingly.

Reply:
Done.